provider/app.py

```diff
@@ -205,7 +226,8 @@ def main():
 
     # Make sure we log to the console
     streamHandler = logging.StreamHandler()
-    formatter = logging.Formatter('[%(asctime)s] [%(levelname)s] [??] [kafkatriggers] %(message)s')
+    formatter = logging.Formatter('[%(asctime)s] [%(levelname)s] '
```

Comment:
Are all these formatting-only changes necessary? If not, I would suggest removing them since they are just muddying the history of this file. If they are necessary, then what is to prevent us from creating more formatting issues in the future?